### DISCUSSION TITLE
李大维：山上的树发NFT

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#14] [Ideas]
大维老师的一个idea：山上的树发NFT，山上的蜜蜂也要发NFT。这也是“赛博乡村”主题的继续脑洞

[温铁军：实地调查之后发现，可以用资本市场手段发展乡村【践闻录】](https://www.bilibili.com/video/BV1Cf4y1x7xc?p=1&share_medium=iphone&share_plat=ios&share_source=COPY&share_tag=s_i&timestamp=1617512705&unique_k=7x9Lnx)

李大维：山上的蜜蜂也要发NFT... 😉

岳路平：蜜蜂怎么NFT？一个一个蜜蜂的腿绑上环吗[得意]

李大维: 一窝一窝的发，蜂窝的数据资料变成NFT 
500箱蜂窝加传感器（温度，湿度，麦克风）
加上他们的GPS资料
那个做為输入资料，上面用AI产生互动艺术
您来给500窝的蜜蜂搞个展

岳路平：这个互动艺术应该没有那家美术馆可以展览了，地球就是这个美术馆了
这个脑洞可以

李大维: 整个山做美术馆
然后在虚拟世界里面做个数字孪生

**Comment by Lupingyue** [comment]
“密蜜”艺术计划

Ming炫: 搞个有水力发电的地方 实现算力自主 

李大维: 杨雅水库 
@Ming炫 一起来跟500窝蜜蜂搞艺术创作吧？

宝可梦: 加密小蜜蜂nft挖矿基地
简称：一蜜一密[呲牙]

李大维: "一密一蜜" 我们的艺术蜂蜜品牌了
买NFT送蜂蜜

宝可梦: 这个酷了

李大维: 有实体资产绑定的NFT... 
每个瓶子上是那窝蜂产生出来的NFT的地址二维码

宝可梦: 收集“密蜜瓶子”将成为“蜂巢「风潮」”

李大维: 
瓶子都有艺术价值。😉
创造朔源二维码的NFT艺术价值，打造蜜蜂劳动的二级市场。

宝可梦: 北京有一个“气味博物馆”，就是卖瓶子里的“味道”，我们可以搞“盲盒密蜜瓶子机器”

李大维: [强]
"密蜜" 这个名字好

宝可梦: 密蜜盲瓶

Ming炫: 
哈哈哈 值得研究一下

宝可梦: 密蜜黑盒

李大维: @Ming炫 23号有没有空，跟我们一起到山里研究？

宝可梦: 来吧，山里面基

**Comment by Tz-H** [comment]
1. 品牌 —— "一密一蜜" --> "密蜜"
2. 实体资产绑定 —— 对应实体窝蜂所产蜂蜜
3. NFT 创作 —— 通过追踪窝蜂蜜蜂活动（温度/湿度/麦克风/GPS资料等），由AI算法产生唯一互动艺术
4. 盲盒 —— 收集“密蜜瓶子”成为“蜂巢「风潮」”
